.azure-pipelines/templates/maven-release/secure-files.yml

```diff
@@ -0,0 +1,15 @@
+steps:
```

Comment:
add a bolean template parameter "download sign-in files" and add conditions on the last two tasks so we can have a single template.